sdk/python/pulumi_aws/apigateway/account.py

```diff
@@ -177,7 +177,7 @@ def __init__(__self__,
         )])
         cloudwatch_role_policy = aws.iam.RolePolicy("cloudwatchRolePolicy",
             role=cloudwatch_role.id,
-            policy=data["aws_iam_policy_document"]["json"])
+            policy=cloudwatch_policy_document.json)
```

Comment:
Interesting.  Any idea where these changes came from?

Reply:
It's just a change to the example code -- at a glance, it looks internally consistent (`cloudwatch_policy_document` is a variable in scope).